src/components/screens/SchoolSchedule.js

```diff
@@ -22,22 +31,22 @@ class SchoolSchedule extends React.Component {
 	static navigationOptions = ({navigation}) => {
 		return {
 			title: 'Add School Schedule',
-			headerTintColor: '#ffffff',
+			headerTintColor: white,
 			headerTitleStyle: {fontFamily: 'Raleway-Regular'},
 			headerTransparent: true,
 			headerStyle: {
 				backgroundColor: 'rgba(0, 0, 0, 0.2)',
 				marginTop: Platform.OS === 'ios' ? 0 : StatusBar.currentHeight
 			},
-			headerRight: navigation.state.routeName === 'TutorialSchoolSchedule' ? (
+			headerRight: navigation.state.routeName === TutorialSchoolSchedule ? (
 				<IconButton
 					onPress={navigation.getParam('goBack')}
 					icon={({size, color}) => (
 						<Image name='logout'
 							size={size}
 							color={color} />
 					)}
-					color='#ffffff'
+					color={white}
 					size={25}
 				/> 
```

Comment:
Put the closing bracket on the same line of the last prop (we forgot to add it in the coding convention)